onnxruntime/test/providers/cpu/math/gemm_test.cc

```diff
@@ -761,9 +761,10 @@ TYPED_TEST(GemmOpTypedTests, ZeroKWithBias) {
   test.AddInput<TypeParam>("C", {4}, std::vector<TypeParam>(4, static_cast<TypeParam>(1.0f)));
   test.AddOutput<TypeParam>("Y", {4, 4}, std::vector<TypeParam>(16, static_cast<TypeParam>(1.0f)));
 
+  // WebGPU EP doesn't support zero buffer.
   test.ConfigExcludeEps({kCoreMLExecutionProvider, kNnapiExecutionProvider,
                          kDmlExecutionProvider, kDnnlExecutionProvider, kQnnExecutionProvider,
-                         kOpenVINOExecutionProvider})
+                         kOpenVINOExecutionProvider, kWebGpuExecutionProvider})
```

Comment:
I think maybe we can workaround it in your implementation like below:

If (A_shape.size() == 0) {
  has_a = false;
}
if (B_shape.size() == 0) {
  has_b = false;
}
...
if (has_a && has_b)
{
  program.AddInputs({{A, ProgramTensorMetadataDependency::Type},
                     {B, ProgramTensorMetadataDependency::Type}});
}